Contributing.md

```diff
@@ -36,7 +36,7 @@ Before you start off with the contribution,
 
 * Fork this repository by clicking on the fork button on the top of this page.
 <p align="center">
-    <img src="https://raw.githubusercontent.com/bhavesh1129/orphan_support-php/main/assets/img/fork.png"/>
+    <img src="https://github.com/LeeRenJie/idea-lab/blob/cont-img/cont-Img/fork.png"/>
```

Comment:
@LeeRenJie Make an `images` folder in `.github` folder, and then upload all images in that. Then used this images path in `contributing.md` file. Because in future if you delete your forked repo then again this images will not get load.

Reply:
Alright will do the change.

Reply:
All done @sanjay270899

Reply:
@LeeRenJie you still didn't understand my point, but no problem 😅. I will fix it. Thanks for the PR. Keep Contributing!

Reply:
I'm sorry did I get the path wrong? I'm still new to this 😅